simple_ldap_user/simple_ldap_user.module

```diff
@@ -480,8 +480,17 @@ function simple_ldap_user_sync_user_to_drupal($drupal_user) {
   // Load the LDAP user, force a cache reset.
   $ldap_user = SimpleLdapUser::singleton($drupal_user->name, TRUE);
 
-  // Nothing to sync.
-  if (!$ldap_user->exists || (isset($drupal_user->uid) && $drupal_user->uid == 1)) {
+  // Nothing to sync. Check if user is an admin
+  if(isset($drupal_user->uid) && $drupal_user->uid == 1) {
+    return;
+  } //For other users
```

Comment:
Drupal coding standards require the `} elseif ($condition) {` to be on the same line.